pokete_classes/learnattack.py

```diff
@@ -3,10 +3,36 @@
 import time
 import random
 import scrap_engine as se
-from pokete_general_use_fns import std_loop
+from pokete_general_use_fns import std_loop, liner
 from .input import ask_bool, ask_ok
-from .ui_elements import ChooseBox
+from .ui_elements import ChooseBox, Box
 from .detail import Detail
+from .color import Color
+from .attack import Attack
+
+
+class AttackInfo(Box):
+    """Gives information about a certain attack"""
+
+    def __init__(self, attack, p_d, _map):
+        obj = Attack(attack, p_d)
```

Comment:
See comment in attack.py line 4:

```suggestion
    def __init__(self, attack, _map):
        obj = Attack(attack)
```